activities/live-streaming/index.md

```diff
@@ -0,0 +1,53 @@
+---
+type: index
+---
+
+# Live streamed podcast
+
+We use live streams as a tool to communicate. We host a podcast style live stream show called 'Let's talk about public code'.
+
+## Goals
+
+* Become discoverable: create another way for people we would like to know who don't know that we exist or what we do to discover us.
+* Grow our codebase's community.
+* Get in touch directly with key figures of top projects. Possible way in to steward or make them members of the Foundation for Public Code.
+* Get more social media followers since we are going to use those networks for this project.
+
+## Formats, length and schedule
+
+1. Frequency: Ambition to make it at least monthly.
+2. Format: Two different formats - one for codebases, one for the Foundation. For the codebases it is an interview made by two codebase stewards - ideally the community ones - with someone relevant from a codebase. For the Foundation, the format is a conversation between two persons from the foundation. This can be theoretical, about ideas and principles or about news.
+3. Language: English.
+4. Script: We get several questions ready before each episode. This builds familiarity with our approach and can also reveal similarities and differences between codebases.
+5. Length and schedule: Minimum 20 minute livestream during business hours - starting at 14.00 UTC if possible. Promise that the podcast will be live by close of business the next day. Aim to hold this on Tuesdays.
```

Comment:
I don't see a reason for this to be an ordered list. I recommend either making it an unordered list or just removing the list aspect as it is basically separate statements.

Reply:
cc @Ainali @AlbaRoza